Gems/EMotionFX/Code/Source/Integration/Editor/Components/EditorActorComponent.cpp

```diff
@@ -188,6 +192,7 @@ namespace EMotionFX
             , m_attachmentJointIndex(0)
             , m_lodLevel(0)
             , m_actorAsset(AZ::Data::AssetLoadBehavior::NoLoad)
+            , m_excludeFromReflectionCubeMaps(false)
```

Comment:
```suggestion
            , m_excludeFromReflectionCubeMaps(true)
```
It sounds like we want to default to exclude actors, which appears to be the case since this change:
https://github.com/o3de/o3de/pull/14108/files#diff-489dc962e9ade1ed57ce21c859910ec56a83c0192caf65b937d2de6d91d0404eR671